### DISCUSSION TITLE
Enhancing React Native Component Reusability

### DISCUSSION THREAD

**Discussion by mrzachnugent** [founded-labs/react-native-reusables#13] [Announcements]
# Welcome to the discussion board for react-native-reusables! 🚀

Hey everyone,

This repository is all about simplifying your React Native development process by providing reusable components that you can easily adapt to fit your unique project needs. Whether you're working on a small app or a large-scale project, the goal here is to enable seamless customization and streamline your development efforts.

This space is open for all kinds of discussions related to improving component reusability within React Native. Share your thoughts, ideas, suggestions, or any challenges you've faced while tailoring these components to match your specific requirements.

Let's collaborate, brainstorm, and foster a community-driven approach to make react-native-reusables even more versatile and adaptable for everyone.

Looking forward to hearing your experiences and insights!

Cheers,
Zach

**Comment by eznix86** [comment]
I have proposed a [cli](#16), to enhance component reusability, this can be extended to anywhere.
The following commands can be 

```sh
npx rn-reusables init # <---- to initialize project
npx rn-reusables add alert # <------ create a component as we needed
```

**Reply by mrzachnugent** [reply]
Thank you for your suggestions.

If others would like to chip in please checked out @eznix86 discussion [Proposal for rn-reusables-cli · mrzachnugent/react-native-reusables · Discussion #16](https://github.com/mrzachnugent/react-native-reusables/discussions/16)
https://github.com/mrzachnugent/react-native-reusables/discussions/16

**Comment by eznix86** [comment]
Another proposal is a guide/website for users of this projects. Read more at #17

**Reply by mrzachnugent** [reply]
If others would like to chip in please checked out @eznix86 discussion [Proposal for a website, it can be hosted at github.io · mrzachnugent/react-native-reusables · Discussion #17](https://github.com/mrzachnugent/react-native-reusables/discussions/17)
https://github.com/mrzachnugent/react-native-reusables/discussions/17
